eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -0,0 +1,24 @@
+function Get-NamepspacesFromDll($dllPath) {
```

Comment:
Any reason not to put this directly in Language-Settings.ps1?

Reply:
It will be reused for toc generation in docindex. We collect all toc related function in this file.